src/components/files/WebDavUrl.vue

```diff
@@ -80,6 +80,11 @@ export default defineComponent({
 			webdavUrlCopied: false,
 		}
 	},
+	computed: {
+		webdavInfo() {
+			return t('simplesettings', '{productName} fully supports the WebDAV protocol and you can connect and synchronise with your {productName} files via WebDAV. Use this address to access your files via WebDAV.', { productName: window.oc_defaults.productName })
```

Comment:
todo: here `oc_defaults.productName` is still `Nextcloud`.

to fix that please add thise code to the `themes/nc-ionos-theme/defaults.php`

```php
	/**
	 * Returns the short name of the software (used e.g in the translation files of the settings app)
	 * @return string title
	 */
	public function getProductName(): string {
		return 'HiDrive Next';
	}
```

ideally we'd not have 4 functions that return `HiDrive Next` as a string, so maybe let those functions return a const

Reply:
implemented with https://github.com/IONOS-Productivity/nc-simplesettings/pull/83/commits/810eaa9b90f1faf02de36d4dc3d7bbc105c7ac2a